api4/system.go

```diff
@@ -1008,3 +1016,88 @@ func checkHasNilFields(value any) bool {
 
 	return false
 }
+
+func getEventsTopics(c *Context, w http.ResponseWriter, r *http.Request) {
+	// parameter withSchema is to be parsed
+	// restrict only for admins
+	if !c.IsSystemAdmin() {
+		c.Err = model.NewAppError("getEventsTopics", "api.get_events_topics.permission_error", nil, "", http.StatusBadRequest)
```

Comment:
```suggestion
		c.Err = model.NewAppError("getEventsTopics", "api.get_events_topics.permission_error", nil, "", http.StatusForbidden)
```

This applies to similar occasions as well.